plugins/ipick.yaml

```
@@ -0,0 +1,34 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: ipick
+spec:
+  version: v1.0.4
+  homepage: https://github.com/similarweb/kubectl-ipick
+  shortDescription: A kubectl wrapper for interactive resource selection.
+  description: |
+    This plugin is a smart wrapper around kubectl, which lets you pick 
+    the resource to act on. For example, kubectl ipick exec will 
+    show a list of pods and execute kubectl exec <selected-pod>.
```

```suggestion
    the resource to act on. For example, `kubectl ipick exec` will 
    show a list of pods and execute `kubectl exec <selected-pod>`.
```
nit: It would improve readability to enclose the commands in backticks -- even though this is just dumped to the command line without any markup.

---

changed. thanks !